src/components/Icon/icons/IconChevronRight.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC, ReactElement } from "react";
+
+import { SvgIconProps } from "../Icon";
+
+const IconChevronRight: FC<SvgIconProps> = ({
+  className = "",
+}): ReactElement => (
+  <svg viewBox="0 0 24 24">
+    <path
+      d="M9.99997 6L8.58997 7.41L13.17 12L8.58997 16.59L9.99997 18L16 12L9.99997 6Z"
+      fill="white"
```

Comment:
I think this should be `currentcolor` which means that the fill is the same as the CSS text colour. This makes it easier to change the colour of the icons (e.g. in light mode). P.S. I learned this from @piersss' code, thanks!